tech-docs/core-contracts/lending-pool/protocol-tokens.md

```diff
@@ -0,0 +1,33 @@
+# Protocol Tokens
```

Comment:
We should call out these are purely accounting structures. Calling this Protocol Accounting may be a better option

Reply:
Yeah I think just use the word token less, tokenized should be cut entirely